fem/fespace.hpp

```diff
@@ -367,7 +367,7 @@ class FiniteElementSpace
        All elements will use the same IntegrationRule, @a ir as the target
        quadrature points. */
    const QuadratureInterpolator *GetQuadratureInterpolator(
-      const IntegrationRule &ir) const;
+      const IntegrationRule &ir, const DofToQuad::Mode = DofToQuad::FULL) const;
```

Comment:
I think it is better to keep the interface simpler and NOT add the `DofToQuad::Mode` parameter here -- the evaluation mode (tensor or non-tensor) of the returned `QuadratureInterpolator` can be set by a call to the returned object itself.

Reply:
`done`